httpserver/http_connection.cpp

```diff
@@ -257,6 +257,36 @@ void connection_t::read_request() {
     http::async_read(socket_, buffer_, request_, on_data);
 }
 
+bool connection_t::verify_signature() {
+    const std::vector<std::string> keys = {LOKI_SENDER_SNODE_PUBKEY,
```

Comment:
It makes me sad that we need to allocate a vector just to check a well-known headers. Could be use some template magic here?

Reply:
I'll see what I can do :)